docs/_includes/common/success-message.html

```diff
@@ -0,0 +1,71 @@
+<div id="success-message">
+    <h2><span class="secondary-text">@ebay/skin/</span>success-message</h2>
+
+    <p>A success-message is a message of celebration used when a user has completed something important. Its higher energy than a confirmation notice and is always the major focus of the page.</p>
+    <p>It can be used within a dialog, or as a full screen in mobile applications.</p>
+
+    <h3>With an action button</h3>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <section aria-labelledby="success-message-title" class="success-message" role="region">
+                <div class="success-message__status"></div>
+                <h2 id="success-message-title" class="success-message__title">Your first order has been placed!</h2>
+                <div class="success-message__content">You'll get a confirmation email soon. The rest of your items are now ready to checkout.</div>
+                <button class="success-message__button">Continue to Order 2</button>
+            </section>
+        </div>
+    </div>
+    {% highlight html %}
+<section aria-labelledby="success-message-title" class="success-message" role="region">
+    <div class="success-message__status"></div>
+    <h2 id="success-message-title" class="success-message__title">Your first order has been placed!</h2>
+    <div class="success-message__content">You'll get a confirmation email soon. The rest of your items are now ready to checkout.</div>
+    <button class="success-message__button">Continue to Order 2</button>
+</section>
+    {% endhighlight %}
+
+    <h3>Without button</h3>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <section aria-labelledby="success-message-title" class="success-message" role="region">
+                <div class="success-message__status"></div>
+                <h2 id="success-message-title" class="success-message__title">Your first order has been placed!</h2>
+                <div class="success-message__content">You'll get a confirmation email soon. The rest of your items are now ready to checkout.</div>
```

Comment:
Can we wrap the content in a `p` tag? I think this is the last thing, then we are good to go.